docs/admin/daemons.md

```diff
@@ -209,9 +209,9 @@ all or certain hosts, and when it needs to start before other pods.
 
 ## Caveats
 
-DaemonSet is part of the experimental API group, so it is not subject to the same compatibility
+DaemonSet is part of the `extensions` API group, so it is not subject to the same compatibility
```

Comment:
"not subject to the same compatibility guarantees" needs fixing, but I don't have a concrete suggestion off the top of my head. possibly overlaps w/ @erictune's doc changes.